### ISSUE TITLE
approx_top_k is not recognized as an aggregation function in Snowflake

### ISSUE THREAD

**Issue by yogevyuval** [tobymao/sqlglot#2449]
**Fully reproducible code snippet**

```python
from sqlglot import Dialects, exp, parse_one

query = "SELECT approx_top_k(field, 5) FROM  some_table"
parsed_query = parse_one(query, read=Dialects.SNOWFLAKE.value)
agg_funcs = parsed_query.find_all(exp.AggFunc)
has_agg_funcs = any(agg_funcs) # getting False, expected True

```

**Official Documentation**
https://docs.snowflake.com/en/sql-reference/functions/approx_top_k

**Comment by GeorgeSittas** [comment]
@yogevyuval I fixed this in the linked PR, but will close similar issues in the future as not planned if they're not directly related to transpilation errors, since it's not high-priority for us. Will happily accept PRs for them though, if you'd like to contribute.

**Comment by yogevyuval** [comment]
> @yogevyuval I fixed this in the linked PR, but will close similar issues in the future as not planned if they're not directly related to transpilation errors, since it's not high-priority for us. Will happily accept PRs for them though, if you'd like to contribute.

Understood - and thank you!

**Comment by yogevyuval** [comment]
@GeorgeSittas Btw is there some kind of a release schedule? When can we expect a new version to come out? No rush just trrying to understand

**Comment by GeorgeSittas** [comment]
Hey, we don't have a specific schedule that we follow but the release cadence is very frequent and so this should be live soon, maybe some time today even.

**Comment by GeorgeSittas** [comment]
Feel free to join our [Slack](https://tobikodata.com/slack) to discuss future issues and/or possibly contributing if you're interested.

**Comment by GeorgeSittas** [comment]
Just released v18.17.0, should be live in a couple of minutes
